src/bp/ui-shared/tsconfig.json

```diff
@@ -20,7 +20,7 @@
     "paths": {
       "~/*": ["*"],
       "botpress/sdk": ["../../sdk/botpress.d.ts"],
-      "common/*": ["../../../../out/bp/common/*"],
+      "common/*": ["../../common/*"],
```

Comment:
what's this?

Reply:
To access common stuff in ui-shared, technically common is shared between backend/frontend, modules, studio, core, etc